packages/ses/ERRORS.md

```diff
@@ -101,7 +101,7 @@ also use a comma expression.
 (0, compartment.import)(specifier);
 ```
 
-## SES2: Possible HTML comment rejected
+## Possible HTML comment rejected
```

Comment:
With the links being to anchors in a page, when you follow a link it is not at all clear what section you're supposed to look at. Perhaps each of these targets should be its own standalone explanation within a subdirectory of such explanations.

Reply:
Changed each to a separate standalone page with an `error-codes` subdirectory. The page filename is now a pleasingly symbolic identifier without putting these identifiers in the programmer's face. Resolving.